jenkins/run-pytests

```diff
@@ -6,7 +6,6 @@
 # NOTE:  Before invoking this script, the various bits of Python setup should
 #        have been performed, e.g., via `source jenkins/python-setup.sh`.
 
-prog="$(basename "${0}")"
```

Comment:
Since this is now the only change to this file (other than adding a blank line at line 24), do you still want to include this file in the PR?